content/blog/2019/05/2019-05-30-becoming-contributor-newbie-tickets.adoc

```diff
@@ -0,0 +1,254 @@
+of approvals necessary might vary, depending on the complexity of the change or the area of Jenkins core that it affects.
+
+After the necessary approvals have been received, a Jenkins core maintainer will set the PR as  `ready-for-merge`, which
+will lead to it being merged into the master branch when one of the following releases are being prepared.
+
+In my case, I received a review by Daniel (the reporter of the ticket and one of my “desired reviewers”) the very day I
+submitted the PR (April 14th). He made several very useful suggestions, which led to changes from my side. After those
+changes, Daniel made minor remarks and my PR got another review, which was its first approval. After a week had passed
+without further news, I added the remaining minor suggestions from Daniel and a few days later received another approval,
+to which Daniel’s final approval was added, leading the PR to be labeled `ready-for-merge`, being later merged the same
+day (April 26th).
+
+.Screenshot of the final state of the PR, after being merged
+image::/images/post-images/2019-05-30-becoming-contributor/pr-merged.png[]
+
+=== Release
+
+For every new release, repository maintainers will select a set of PRs that have already been labeled `ready-for-merge`,
+merge them to master, prepare changelogs (often using the suggestions included in the PRs by the authors) and proceed with
+the creation of the new release. There is no additional action required from Pull Request authors at this point.
+
+Every week a new version of Jenkins is released, so when your PR is merged, your changes will--most likely--become part
+of the following weekly release of Jenkins.
+
+Eventually, your changes will also reach the https://jenkins.io/download/lts/[Long-term support (LTS) release], which is
+a different release line, aimed for more conservative users. This release line gets synced with the weekly release by
+picking, every 12 weeks, a relatively recent weekly release as baseline for the new LTS release. In between, intermediate
+LTS releases are created only to include important bug fixes, cherry-picked from the weekly releases. New features are
+typically delayed until the next baseline for the LTS release is defined.
+
+Regarding the example described in this post, *it was released in Jenkins `2.175`* (weekly release), soon after being
+merged. And will probably be included in the next LTS, which should be released next month (June 2019).
+
+== Done!
+
+And that’s it! We have now covered the whole lifecycle of a new proposed change to Jenkins core. We have reviewed the
+process from the very beginning, picking a ticket from the Jenkins issue tracker; all the way to the end, having our
+change released in a new Jekins version.
+
+If you have never contributed but are willing to do so, I hope this article motivates you to go back to the list of
+`newbie-friendly` tickets, find one that looks interesting to you, and follow the steps described above, until you see
```

Comment:
Make the concluding reference to `newbie-friendly tickets` clickable like the first reference is.  As they heed the call to action, they can immediately click the link and identify how they can help.
```suggestion
https://issues.jenkins-ci.org/browse/WEBSITE-625?jql=labels%20%3D%20newbie-friendly[`newbie-friendly` tickets], find one that looks interesting to you, and follow the steps described above, until you see
```